### ISSUE TITLE
Better error messages on failure

### ISSUE THREAD

**Issue by humblearner** [StackStorm/st2web#451]
1. Core packs: `core`, `default` packs cannot be uninstalled.

the failure is not user friendly: 
![image](https://user-images.githubusercontent.com/11877841/34022731-c09c865c-e0f5-11e7-9270-34ef066bb899.png)

CLI:

```bash
$ st2 pack remove core

	[  failed   ] unregister packs

id: 5a332817c5ff1b0d739d7fce
action.ref: packs.uninstall
parameters:
  packs:
  - core
status: failed
result_task: unregister packs
result:
  exit_code: 1
  result: None
  stderr: "Traceback (most recent call last):
  File "/opt/stackstorm/st2/lib/python2.7/site-packages/st2common/runners/python_action_wrapper.py", line 276, in <module>
    obj.run()
  File "/opt/stackstorm/st2/lib/python2.7/site-packages/st2common/runners/python_action_wrapper.py", line 169, in run
    output = action.run(**self._parameters)
  File "/opt/stackstorm/packs/packs/actions/pack_mgmt/unload.py", line 59, in run
    raise ValueError('Unregister includes an unregisterable pack - %s.' % (names))
ValueError: Unregister includes an unregisterable pack - core.
"
  stdout: ''
error: Traceback (most recent call last):
  File "/opt/stackstorm/st2/lib/python2.7/site-packages/st2common/runners/python_action_wrapper.py", line 276, in <module>
    obj.run()
  File "/opt/stackstorm/st2/lib/python2.7/site-packages/st2common/runners/python_action_wrapper.py", line 169, in run
    output = action.run(**self._parameters)
  File "/opt/stackstorm/packs/packs/actions/pack_mgmt/unload.py", line 59, in run
    raise ValueError('Unregister includes an unregisterable pack - %s.' % (names))
ValueError: Unregister includes an unregisterable pack - core.

traceback: None
failed_on: unregister packs
start_timestamp: Fri, 15 Dec 2017 01:40:39 UTC
end_timestamp: Fri, 15 Dec 2017 01:40:41 UTC
+--------------------------+---------------------+------------------+--------------+------------------+
| id                       | status              | task             | action       | start_timestamp  |
+--------------------------+---------------------+------------------+--------------+------------------+
| 5a332817c5ff1b0c0bb4c2be | failed (2s elapsed) | unregister packs | packs.unload | Fri, 15 Dec 2017 |
|                          |                     |                  |              | 01:40:39 UTC     |
+--------------------------+---------------------+------------------+--------------+------------------+
[vagrant@centos67 ~]$
```

**Comment by christopherjbaker** [comment]
It appears that the old and new app are behaving the same in this case, though I do agree that it is poor UX. Since the current goal is feature parity, I am going to leave it as-is for now, but I will leave this open to be tackled as a feature request.

In all cases where something fails, it simply says that same notification.  I think the reason here is because the only result that comes back from a failure is the CLI stderr, which isn't terribly useful in a small notification. In this case, the last line is the significant part. I don't think it would be too difficult to make a method that can extract the relevant info for the notification (I imagine other errors won't follow this exact structure, so some effort here).

```
Traceback (most recent call last):
  File \"/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2common/runners/python_action_wrapper.py\", line 259, in <module>
    obj.run()
  File \"/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2common/runners/python_action_wrapper.py\", line 155, in run
    output = action.run(**self._parameters)
  File \"/opt/stackstorm/packs/packs/actions/pack_mgmt/unload.py\", line 59, in run
    raise ValueError('Unregister includes an unregisterable pack - %s.' % (names))
ValueError: Unregister includes an unregisterable pack - core.
```

**Comment by christopherjbaker** [comment]
see also, #410

**Comment by christopherjbaker** [comment]
This has been fixed in the [react rewrite](https://github.com/StackStorm/st2web/pull/420), which should be included with 2.6, though this feature may not be included with the initial release.
